### DISCUSSION TITLE
Add new permission

### DISCUSSION THREAD

**Discussion by Darkseno** [Zrips/CMI#7484] [Feature Requests]
Hello,

There are already permissions jobs.maxfurnaces.[amount] & jobs.maxbrewingstands.[amount] 

Would it be possible to add the same permissions for Smoker and blastfurnaces? It would be very interesting for my ranks and jobs please.

Have a nice day

**Comment by mrfloris** [comment]
This is the repository for the zrips-CMI plugin, not jobs-reborn, based on the info from your post, can you perhaps repost it here: <https://github.com/Zrips/Jobs/issues/new?assignees=&labels=New+Feature+Suggestion&template=feature_request.md&title=> thank you.
